fab/fab/operations/supervisor.py

```diff
@@ -106,10 +112,15 @@ def set_celery_supervisorconf():
             conf_destination_filename = 'supervisor_celery_worker_{}_{}.conf'.format(
                 comma_separated_queue_names, worker_num)
 
+            worker_name = get_celery_worker_name(env.env_name,
```

Comment:
yeah in all of these places where you're using `env.env_name` (fab) or `args.environment` (commcare-cloud) to generate the supervisor service name, you should use `env.deploy_env` (fab) or `environment.meta_config.deploy_env` (commcare-cloud) instead.

Reply:
responded earlier [here](https://github.com/dimagi/commcare-cloud/pull/1475#discussion_r176038699)